index.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Brahim's Portfolio Page</title>
+    <!-- css links -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+</head>
+<body>
+   <header id="header">
+       <h1 class="headertitle">Jhon Doe</h1>
+<nav>
+    <img src="template5Images/Union.svg" alt="button">
+</nav>
+   </header>
+   <main class="main">
+  <section class="section1">
+  <div class="Names">
+      <h2 class="h2">Brahim Didi</h2>
```

Comment:
- [ ] This being the main title, change it to a `h1` tag.

Reply:
those are easy changes that wil require me to make a new request that's really not nice from you